x-pack/plugins/secops/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -359,11 +356,7 @@ const getAuthenticationColumns = (startDate: number): Array<Columns<Authenticati
               ) : isNil(get('lastSuccess.host.id', node)) ? (
                 <>{hostName}</>
               ) : (
-                <EuiLink
-                  href={`#/link-to/hosts/${encodeURIComponent(node.lastSuccess!.host!.id!)}`}
-                >
-                  {hostName}
-                </EuiLink>
+                <HostDetailsLink hostId={node.lastSuccess!.host!.id!}>{hostName}</HostDetailsLink>
```

Comment:
My opinion here. Nothing needs to change if you don't want it to but we have had conversations in the past about `host.id` probably being required. If it's null this is going to get goofy quick.

My suggestion since I would like to minimize the TypeScript `!` is to use a get above this for `hostId` and check for its existence:

And try not to get fall into the bang trap that is turning off more and more of our types. Later if we want to make `host.id` required that would be fine as well.

No changes, just suggestions to not rely on bang too much. I am trying to avoid it everywhere I can and protect the UI with null guards where I can.

```ts
      const hostName: string | null = get('lastSuccess.host.name', node);
      const hostId: string | null = get('lastSuccess.host.id', node);
      if (hostName != null && hostId != null) {
        return (
          <DraggableWrapper
            dataProvider={{
              and: [],
              enabled: true,
              id: escapeDataProviderId(`authentications-table-${node._id}-lastSuccess-${hostName}`),
              name: hostName,
              excluded: false,
              kqlQuery: '',
              queryMatch: {
                displayField: 'host.name',
                displayValue: hostName,
                field: 'host.id',
                value: hostId,
              },
              queryDate: {
                from: startDate,
                to: Date.now(),
              },
            }}
            render={(dataProvider, _, snapshot) =>
              snapshot.isDragging ? (
                <DragEffects>
                  <Provider dataProvider={dataProvider} />
                </DragEffects>
              ) : isNil(get('lastSuccess.host.id', node)) ? (
                <>{hostName}</>
              ) : (
                <HostDetailsLink hostId={hostId}>{hostName}</HostDetailsLink>
              )
            }
          />
```